plinth/templates/uninstall.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% comment %}
```

Comment:
I don't think removing the left menu extra to give the uninstall process a completely different look is the best way to go. Since progress should be shown for uninstall, the user should be able to navigate to other parts of the Plinth during uninstall and be able to come back to see the progress. Also during uninstallation, how does the application page look like?

We should make the uninstall process look very similar to installation process to avoid these problems. After the uninstall conformation page is shown and the uninstall is happening, we can redirect the user to the application page where the progress is shown.